File: AvernusGames_Front-End/src/app/models/Fornecedor.ts
import { Endereco } from "./Endereco";

export class Fornecedor {
    id: number = 0;
    cnpj: string = '';
    nome: string = '';
    email: string = '';
    telefone: string = '';
    endereco: Endereco = new Endereco();
    enderecoId: number = 0;
  }
  

File: AvernusGames_Front-End/src/models/Fornecedor.cs
namespace Avernus_Games_Store.src.models
{
    public class Fornecedor
    {
        public int Id { get; set; }
        public string? CNPJ { get; set; }
        public string? Nome { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public Endereco? Endereco { get; set; }

        public Fornecedor() { }

        public Fornecedor(string cnpj, string nome, string email, string phone, Endereco endereco)
        {
            CNPJ = cnpj;
            Nome = nome;
            Email = email;
            Phone = phone;
            Endereco = endereco;
        }
    }
}